tiledb/type/range/range.h

```diff
@@ -451,10 +451,9 @@ Status check_range_is_subset(const Range& superset, const Range& range) {
  *
  * @param range The range to check.
  */
-template <
```

Comment:
Does this need to be removed?

Reply:
You mean the function? No, it is generic and is being used by the non-generic `check_range_is_valid(const Range&, DataType)`.